app/src/main/res/values/strings.xml

```diff
@@ -596,4 +596,9 @@ Upload your first media by tapping on the add button.</string>
   <string name="wallpaper_set_unsuccessfully">Something went wrong. Could not set the wallpaper</string>
   <string name="setting_wallpaper_dialog_title">Set as Wallpaper</string>
   <string name="setting_wallpaper_dialog_message">Setting Wallpaper. Please wait…</string>
+
+  <string name="cannot_open_location_settings">Failed to open location settings. Please turn on location and set it to High Accuracy</string>
```

Comment:
I just wanted to share another thing I've been wondering about the usage of the word "High Accuracy" for some time now. Apologies before hand if I'm being overly pedantic. Personally, due to the diversity of Android devices out there I don't think we could even use "High Accuracy" reliably. For instance, my device reads out the parameters it uses in each location mode:
![image](https://user-images.githubusercontent.com/12448084/76451511-881d9980-63f5-11ea-94cc-b7d42deca8c1.png)


Pixel has a totally different approach:
![Screenshot_1583951313](https://user-images.githubusercontent.com/12448084/76451434-691f0780-63f5-11ea-8846-4fc91043f6e7.png)

So, I'm not sure if that would be a best choice. But I don't think we could do anything better about it too. Let me know if there are better suggestions.

Reply:
> I just wanted to share another thing I've been wondering about the usage of the word "High Accuracy" for some time now. Apologies before hand if I'm being overly pedantic. Personally, due to the diversity of Android devices out there I don't think we could even use "High Accuracy" reliably. For instance, my device reads out the parameters it uses in each location mode:
> 
> Pixel has a totally different approach:
> 
> So, I'm not sure if that would be a best choice.
> 
> That said, I don't think we could do anything better about it too. Let me know if there are better suggestions.

Yes, this is a valid point. Also, if we show a long and descriptive toast, it will soon disappear before they can read all of it.
I will let you know if anything good comes to mind.

Reply:
![location](https://user-images.githubusercontent.com/46349662/76640247-972b5580-6575-11ea-9f4f-b981840fcc0b.jpg)

@sivaraam How does this look? Can we replace the `Please set location mode to High Accuracy` with the above-shown toast?